pkgs/development/libraries/ogre/default.nix

```diff
@@ -8,11 +8,11 @@
 , nvidia_cg_toolkit }:
 
 stdenv.mkDerivation {
-  name = "ogre-1.9.0";
+  name = "ogre-1.9-hg-20160322";
```

Comment:
Use the name that makes the most sense to you. The important thing is that the version sorts properly (that is, `old < new`).